Application/Users/AcceptOrRemoveFriendRequest/AcceptOrRemoveFriendRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Domain;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Users.AcceptOrRemoveFriendRequest
+{
+    public class AcceptOrRemoveFriendRequest : IRequest
+    {
+        public User LoggedUser { get; set; }
+        public int idFriend { get; set; }
+        public bool removeFriendRequest { get; set; }
```

Comment:
You can rename the class to UpdateFriendRequest and the bool variable removeFriendRequest to accepted - by default it will be false.
Needs a little bit of refactorisation.

Reply:
Updated code.
Check "renamed AcceptOrRemoveFriendRequest to UpdateFriendRequest, renamed..." commit